src/cli.ts

```diff
@@ -1,12 +1,17 @@
-import commander from "commander";
+import { Command, Option } from "commander";
 import upload from "./upload";
 import download from "./download";
 import collect from "./collect";
 import deleteBranch from "./delete-branch";
+import config from "./config";
 
 const main = async (argv: string[]) => {
-  const program = new commander.Command();
+  const program = new Command();
   const version = require("../package.json").version;
+  const branchOption = new Option(
+    "-b, --branch-name [string]",
+    "the Crowdin branch name where to sync the translations"
```

Comment:
```suggestion
    "the name of the branch to be created in Crowdin"
```

Reply:
Think you can ignore this one, as it won’t apply when used in the `download` command.